asapdiscovery-alchemy/asapdiscovery/alchemy/schema/fec.py

```diff
@@ -225,9 +335,14 @@ class _FreeEnergyBase(_SchemaBase):
         MultiStateSimulationSettings(
             equilibration_length=1.0 * OFFUnit.nanoseconds,
             production_length=5.0 * OFFUnit.nanoseconds,
+            time_per_iteration=2.5 * OFFUnit.picoseconds,
```

Comment:
I would keep this at the default value while we look into large differences from the test you ran.